miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryDilutionRestController.java

```diff
@@ -139,4 +140,11 @@ public DilutionDto updateDilution(@PathVariable Long dilutionId, @RequestBody Di
     return jQueryBackend.get(request, response, null, PaginationFilter.pool(poolId));
   }
 
+  @RequestMapping(value = "query", method = RequestMethod.POST, produces = { "application/json" })
+  @ResponseBody
+  public List<DilutionDto> getSamplesInBulk(@RequestBody List<String> names, HttpServletRequest request, HttpServletResponse response,
```

Comment:
rename method `getDilutionsInBulk`. Similar on Library and Pool RestControllers too